### ISSUE TITLE
external GeoNode reference uses GPL

### ISSUE THREAD

**Issue by tomkralidis** [geopython/pycsw#310]
cc @kalxas 

The code at https://github.com/geopython/pycsw/blob/master/pavement.py#L313 references what is now https://github.com/GeoNode/geonode/blob/master/pavement.py#L598 (i.e. `pavement.py`, `def kill`).

GeoNode is [GPL](https://github.com/geonode/geonode#gpl-license)

**Comment by kalxas** [comment]
Good catch!
Yes this is an issue. We can not re-license this code to MIT.

**Comment by tomkralidis** [comment]
@kalxas I can remove that code and replace with this instead:

```python
def kill_process(procname, scriptname):
    """kill WSGI processes that may be running in development"""

    # from http://stackoverflow.com/a/2940878
    import subprocess, signal
    p = subprocess.Popen(['ps', 'aux'], stdout=subprocess.PIPE)
    out, err = p.communicate()

    for line in out.splitlines():
        if procname in line and scriptname in line:
            pid = int(line.split()[1])
            info('Stopping %s %s %d' % (procname, scriptname, pid))
            os.kill(pid, signal.SIGKILL)
```

Would that work?  Will apply to master and 1.10.

**Comment by tomkralidis** [comment]
Fixed in master and 1.10.
